Kitodo/src/main/webapp/WEB-INF/templates/includes/system/migration.xhtml

```diff
@@ -104,6 +111,16 @@
                 update="systemTabView:migrationForm"/>
         </h:panelGroup>
 
+        <h:panelGroup id="ldapManagerPasswordsMigration" rendered="#{MigrationForm.ldapManagerPasswordsMigrationRendered}">
+            <div style="margin: 10px; width: 50%;"><h:outputText value="#{msgs.migrateLdapManagerPasswordsDesc}:"/></div>
```

Comment:
Why is there a ":"? The resulting text looks strange:
```
Die Migration verschlüsselt das LDAP Manager Passwort mit dem
"security.secret.ldapManagerPassword" Parameter in der
kitodo_config.properties und überschreibt
das Passwort in der Datenbank. Bestehende Verschlüsselungen werden durch ein erneutes
Ausführen der Migration ignoriert.:
```

Reply:
Exactly, the ":" appears to be a typo and can be removed.